tests/test_editor.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+from app.internal.wysiwyg_editor import clean_html
+
+XXS_ATTACKS = [
```

Comment:
XSS

Reply:
done.
its in the next commit, waiting for you response on the other comment.